Pipfile

```diff
@@ -7,6 +7,8 @@ name = "pypi"
 pytest = "*"
 
 [packages]
+dataclass-csv = "*"
+str2bool = "*"
```

Comment:
Do you think it would be hard to implement this ourselves? I would like to keep the library dependency free and only use what's available in the standard library.

Reply:
We could use this: https://docs.python.org/3.7/distutils/apiref.html?highlight=strtobool#distutils.util.strtobool

Reply:
Nice! Yeah, I agree that it would be better to use this built-in method. As you noted below, we could handle the `ValueError` and return `None` in those cases if that sounds good (to mimic the behavior of the `str2bool` package. I'll make these changes this evening when I get home.